services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py

```diff
@@ -2,47 +2,52 @@
 from typing import Awaitable, Callable
 
 from fastapi import FastAPI
-from servicelib.background_task import start_periodic_task, stop_periodic_task
-from servicelib.redis_utils import exclusive
-from settings_library.prometheus import PrometheusSettings
+from servicelib.logging_utils import log_catch, log_context
+from servicelib.rabbitmq import RabbitMQClient, RabbitSettings
 
 from .core.settings import ApplicationSettings
-from .modules.redis import get_redis_client
-from .resource_tracker_core import collect_container_resource_usage_task
-
-_TASK_NAME = "periodic_prometheus_polling"
+from .modules.rabbitmq import get_rabbitmq_client
 
 _logger = logging.getLogger(__name__)
 
 
+async def _subscribe_to_rabbitmq(app) -> None:
+    with log_context(_logger, logging.INFO, msg="Subscribing to rabbitmq channel"):
+        rabbit_client: RabbitMQClient = get_rabbitmq_client(app)
+
+        # TODO: subscribe to rabbitmq channel
+        print("rabbit_client", rabbit_client)
```

Comment:
remove print

Reply:
2. you can already connect to the correct exchanges:
_RabbitResourceTrackingBaseMessage.get_channel_name()

Reply:
I have one question here, we can discuss it in person.